plugins/mumble_plugin_string.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2005-2017 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_PLUGIN_STRING_H_
+#define MUMBLE_MUMBLE_PLUGIN_STRING_H_
+
+#include <string>
+#include <cstring> // For memset
+
```

Comment:
Whole file is missing comments explaining how these types and their helper functions are meant to work.